TundraTD/Assets/Scripts/SceneManagement/LevelsScene.cs

```diff
@@ -0,0 +1,52 @@
+﻿using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+
+namespace SceneManagement
+{
+    public class LevelsScene : MonoBehaviour
+    {
+        [SerializeField] private RectTransform buttonsHolder;
+        [SerializeField] private Button levelButtonPrefab;
+        private int _levelsAmount;
+        private int _currentLevelBindingIndex = 1;
+        private const int LevelsBindingStartIndex = 2;
+        
+        private void Start()
+        {
+            if (_levelsAmount != 0)
+                return;
+            
+            CountLevelsInBuild();
+            print(_levelsAmount);
```

Comment:
I'd change the 'print' function to Debug.Log, it would make logging more systematized. Also, I guess the debug of the file is ended, so this line could be removed